components/browser/thumbnails/src/main/java/mozilla/components/browser/thumbnails/utils/ThumbnailDiskCache.kt

```diff
@@ -24,7 +23,6 @@ class ThumbnailDiskCache {
     private var thumbnailCache: DiskLruCache? = null
     private val thumbnailCacheWriteLock = Any()
 
-    @VisibleForTesting(otherwise = VisibleForTesting.NONE)
     internal fun clear(context: Context) {
         getThumbnailCache(context).delete()
```

Comment:
Must have missed this during the previous review, shouldn't this be in a synchronized block as well?

Reply:
Fixed.